pkgs/servers/misc/navidrome/default.nix

```diff
@@ -1,20 +1,26 @@
-{ lib, stdenv, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
+{ lib, stdenv, pkgs, fetchurl, ffmpeg, ffmpegSupport ? true, makeWrapper, nixosTests }:
 
 with lib;
 
 stdenv.mkDerivation rec {
   pname = "navidrome";
-  version = "0.44.1";
+  version = "0.45.1";
 
-  src = fetchurl {
+
+  src = fetchurl (if pkgs.system == "x86_64-linux"
```

Comment:
This is a really bad condition. Not being on x86_64-linux does **not** mean we are on arm even if platforms is set to those both platforms only.

Reply:
Something like this is better?
```nix
src =
    let
      src_x86_64 = {
        url = "https://github.com/deluan/navidrome/releases/download/v${version}/navidrome_${version}_Linux_x86_64.tar.gz";
        sha256 = "sha256-TZcXq51sKoeLPmcRpv4VILDmS6dsS7lxlJzTDH0tEWM=";
      };
      src_aarch64 = {
        url = "https://github.com/deluan/navidrome/releases/download/v${version}/navidrome_${version}_Linux_arm64.tar.gz";
        sha256 = "sha256-Va0DSmemj8hsaywoP6WKo/x+QQzSNwHCpU4VWs5lpbI=";
      };
    in
    fetchurl (
      if system == "x86_64-linux" then src_x86_64
      else if system == "aarch64-linux" then src_aarch64
      else throw "Navidrome is supported only for \"x86_64-linux\" and \"aarch64-linux\" platforms"
    );
```